Docs/Configuration.tex

```diff
@@ -4777,10 +4777,10 @@ \subsection{Generic Properties}\label{platforminfogeneric}
 
   \begin{itemize}
     \item \texttt{FW\_FEATURE\_SUPPORTS\_CSM\_LEGACY\_MODE} (\texttt{0x1})
-    - Without this bit it is not possible to reboot to Windows installed on
+    - Without this bit, it is not possible to reboot to Windows installed on
       a drive with EFI partition being not the first partition on the disk.
```

Comment:
Should add that recent Windows 10 installations are not affected. @khronokernel could you tell us when was that fixed?

Reply:
Left alone pending indication from @khronokernel

Reply:
We will address it later.